src/optimizer/OptimizerUtils.cpp

```diff
@@ -430,11 +430,55 @@ Value OptimizerUtils::boundValueWithMin(const meta::cpp2::ColumnDef& col, const
         case Value::Type::PATH: {
             DLOG(FATAL) << "Not supported value type " << type
                         << "for index.";
-            return Value(NullType::BAD_TYPE);
+            return Value::kNullBadType;
         }
     }
     DLOG(FATAL) << "Unknown value type " << static_cast<int>(type);
-    return Value(NullType::BAD_TYPE);
+    return Value::kNullBadType;
+}
+
+Value OptimizerUtils::normalizeValue(const meta::cpp2::ColumnDef& col, const Value& v) {
+    auto type = SchemaUtil::propTypeToValueType(col.get_type().get_type());
```

Comment:
Need check does `type` equal to `v.type()`?

Reply:
> Need check does `type` equal to `v.type()`?

Not need check at here, the type checked in another method . such as :
```
(root@nebula) [s]> lookup on tag1 where tag1.col1 == "row1_c1"
+------+
| _vid |
+------+
| "v1" |
+------+
Got 1 rows (time spent 1605/1987 us)

Wed, 04 Nov 2020 18:44:55 CST

(root@nebula) [cbs]> lookup on tag1 where tag1.col1 == 1
[ERROR (-12)]: SemanticError: expression error : tag1.col1

Wed, 04 Nov 2020 18:44:59 CST

(root@nebula) [s]> 
```